tests/run-make/issue-106755/Makefile

```diff
@@ -0,0 +1,4 @@
+include ../../run-make-fulldeps/tools.mk
+
+all:
+	$(RUSTC) main.rs -Ztranslate-lang=en_US 2>&1 | diff - out.stderr
```

Comment:
Can't this just be a UI test with a `compile-flags` header? r=me after that change.

Reply:
Actually it can be, I tested [here](https://github.com/rust-lang/rust/actions/runs/4247921510/jobs/7386500640).